### DISCUSSION TITLE
客户端伪造IP时，网站监控-请求日志不显示归属地

### DISCUSSION THREAD

**Discussion by R0A1NG** [1Panel-dev/1Panel#5416] [Q&A]
### 1Panel 版本

v1.10.10-beta.2

### 问题描述

![image](https://github.com/1Panel-dev/1Panel/assets/66672984/3f05ab39-be16-4a12-b0dc-5c329903db83)

顶上那个是没伪造IP正常请求的，IP获取正常，归属地显示正常

底下那个是经过伪造IP的，IP获取正常，归属地显示不正常
![image](https://github.com/1Panel-dev/1Panel/assets/66672984/16d64f62-01fa-40dc-82d2-57e6382dc45b)

### 重现步骤

使用X-Forwarded-For等字段伪造IP访问

**Comment by R0A1NG** [comment]
还有个问题，雷池自动探测网站状态的请求，IP怎么都变unknown了，应该是127.0.0.1啊，我开cdn配置和不开cdn配置都显示unknown
![image](https://github.com/1Panel-dev/1Panel/assets/66672984/474f2ae2-0eb0-4b9b-bdee-e21364f72113)

**Comment by zhengkunwang223** [comment]
好的，我们看一下

**Comment by zhengkunwang223** [comment]
您好 没有复现此问题
